canonical-websites/www.ubuntu.com
# Defer or asynchronously load blocking resources

## Done

- Remove Modernizr
- Add HTML5 shiv inline in head
- Add no-js replacement inline in head
- Place the minified contents of polyfills.js at the bottom of the page, before any real JavaScript is loaded
- Move all other JavaScript to the bottom of the page, below the now inline polyfills
- In core.js, use hasFeature instead of Modernizr.svg (Oper mini supports svg as of Mar 2015)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Check everything works as expected

## Issue / Card

Fixes #656
